domains/skilkry.json

```
@@ -3,7 +3,8 @@
     "username": "skilkry",
     "email": "skilkrymods@gmail.com"
   },
-  "records": {
-    "CNAME": "portfolio-sw5y.vercel.app"
+  "record": {
+    "CNAME": "portfolio-sw5y.vercel.app",
+    "TXT": "vc-domain-verify=skilkry.is-a.dev,a8ecf41372a24237ade1"
```

```suggestion
    "CNAME": "portfolio-sw5y.vercel.app"
```
you can't TXT records for verifying domain in vercel, please read the step 5 of -> https://docs.is-a.dev/guides/vercel/